roles/openshift_metrics/tasks/install_metrics.yaml

```diff
@@ -11,19 +12,23 @@
   loop_control:
     loop_var: include_file
   when: "not {{ openshift_metrics_heapster_standalone | bool }}"
+  run_once: True
 
 - name: Install Heapster Standalone
   include: install_heapster.yaml
   when: "{{ openshift_metrics_heapster_standalone | bool }}"
+  run_once: True
```

Comment:
This will apply to every task in install_heapster.yaml, which makes the run_once: True on generate_heapster_certificates.yaml redundant.  Additionally every task in that include will be run_once, which it looks like is not desired.